.circleci/config.yml

```diff
@@ -1,54 +1,86 @@
 version: 2.1
 jobs:
+  #
+  # Alpine
+  #
   build-alpine:
     executor: docker-executor
     steps:
+      - checkout
+      - setup_remote_docker
       - docker-build:
           image_name: ap-airflow-alpine
           path: 1.10.5/alpine3.10
+      - run:
+          name: "Tag image"
+          command: docker tag ap-airflow-alpine astronomerinc/ap-airflow:1.10.5-alpine3.10
```

Comment:
this block required to have onbuild build from local image instead of pulling

Reply:
@sjmiller609 instead of this extra step, can we change the `docker-build`'s `image_name` to be the actual desired name? Like `ap-airflow:1.10.5-alpine3.10` instead of `ap-airflow-alpine`.

Reply:
there is an issue where the pathing is weird when the name contains a slash (for the save and load from workspace part). we can probably work around that somehow. you are saying without the organization name in it, so that might just work as you are suggesting. I'm good with whatever assuming the pipeline works, but always be cautious for accidentally pulling old images